internal/toolexec/utils/utils.go

```diff
@@ -0,0 +1,35 @@
+package utils
+
+import (
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+// ExitIfError calls os.Exit(1) if err is not nil
+func ExitIfError(err error) {
+	if err != nil {
+		log.Printf("%v", err)
+		os.Exit(1)
+	}
```

Comment:
Style/Optional, although the `log.Printf` is unfortunate because it does not contain a trailing new line, which it should... Also `log.Fatalln` does what you're doing here... :P 

```suggestion
	if err == nil {
	  return
	}
	log.Fatalln(err)
```